src/main/java/in/projecteka/consentmanager/link/discovery/model/patient/response/DiscoveryResult.java

```diff
@@ -17,7 +18,7 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class DiscoveryResult {
     private UUID requestId;
-    private String timestamp;
+    private LocalDateTime timestamp;
```

Comment:
We need to update the timestamp format to LocalDateTime in the HIP before. Otherwise, an error will be thrown in the consent_manager on serialize/deserialize.

Reply:
I don't think it fails because of HIP. it fails because of the way it's serialised within Link in CM. supposed to use

Serializer.java rather it use Own serializer.